src/status_im/ui/screens/browser/views.cljs

```diff
@@ -59,12 +60,13 @@
                        (re-frame/dispatch [:browser.ui/remove-browser-pressed browser-id]))))]
    [toolbar-content url url-original browser url-editing?]])
 
-(defn- web-view-error [_ code desc]
+(defn- web-view-error [_ _ desc]
   (reagent/as-element
    [react/view styles/web-view-error
+    [react/image {:style  {:width 140 :height 140 :margin-bottom 16}
+                  :source {:uri (contenthash/url
+                                 "e3010170122001bbe2f5bfba0305a3bdc2047fddc47ee595a591bdee61de6040ffc2456624e1")}}]
```

Comment:
Cool, this helps with https://github.com/status-im/status-react/pull/10799#issuecomment-644071944 :). But what is the url here?